tutorials/plugins/android/android_plugin.rst

```diff
@@ -55,7 +55,7 @@ The instructions below assumes that you're using Android Studio.
 
 2.  Add the Godot engine library as a dependency to your plugin module:
 
-    -  Download the Godot engine library (godot-lib.x.y.aar)
+    -  Download the Godot engine library (``godot-lib.<version>.<status>.aar``) from the `Godot download page <https://godotengine.org/download>`_ (e.g: ``godot-lib.4.0.0.stable.aar``).
```

Comment:
Actually, the version would be `4.0.stable`. When `patch == 0`, we don't include it in the version string.

Reply:
Resolved in https://github.com/godotengine/godot-docs/pull/3629.